src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,26 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+import { Todo } from '../../types/todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={cn('TodoInfo', {
+        'TodoInfo--completed': todo.completed,
+      })}
+    >
+      <h2 className="TodoInfo__title">
+        {todo.title}
+      </h2>
+
+      <UserInfo userId={todo.userId} />
```

Comment:
```suggestion
      {user && <UserInfo userId={todo.userId} />}
```
As you'll get the user from the todo you'll need to render the `UserInfo` this way.